README.md

```diff
@@ -126,6 +126,7 @@ class MyComponent extends React.Component {
 * `swipingTransitionDuration`: Integer, default `0`
   * transition duration while swiping in milliseconds
 * `slideInterval`: Integer, default `3000`
+* `swipeThreshold`: Integer, default `30`
```

Comment:
Can you add the following to the documentation. Feel free to reword it if its confusing to you.

```
* A percentage of how far the offset of the current slide is swiped to trigger a slide event.
   e.g. If the current slide is swiped less than 30% to the left or right, it will not trigger a slide event.
```